src/Model/ModelManagerInterface.php

```diff
@@ -117,13 +117,12 @@ public function getParentFieldDescription($parentAssociationMapping, $class);
 
     /**
      * @param string $class
-     * @param string $alias
      *
      * @return ProxyQueryInterface
      *
      * @phpstan-param class-string $class
      */
-    public function createQuery($class, $alias = 'o');
+    public function createQuery($class);
```

Comment:
Doing this directly is a BC-break. It should be done on the master branch instead, with an upgrade note.

Reply:
Hmmm I don't think it is BC-break in this case, I think you can remove it since it is the last argument and it has a default value.

Reply:
I don't know if this attempts against LSP in some way, but from the PHP implementation, this change seems valid (https://3v4l.org/NLTIK).